install_config/master_node_configuration.adoc

```diff
@@ -802,21 +802,6 @@ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/openp
 [[master-node-config-advanced-audit]]
 === Advanced Audit
 
-[IMPORTANT]
```

Comment:
@mpbarrett & @pweil-  this is dropping the tech-preview from audit.

Reply:
Nice!